libpimeval/src/libpimeval.h

```diff
@@ -30,6 +30,7 @@ enum PimDeviceEnum {
   PIM_DEVICE_BITSIMD_H,
   PIM_DEVICE_FULCRUM,
   PIM_DEVICE_BANK_LEVEL,
+  PIM_DEVICE_UNKNOWN,
```

Comment:
We can use PIM_DEVICE_NONE for unknown cases

Reply:
@deyuan Thank you for identifying this issue. Will be fixed.